packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  LinkLibrary,
+  thunderstoreLinkProps,
+  ThunderstoreLinkProps,
+} from "@thunderstore/components";
+import NextLink from "next/link";
+import React, { forwardRef } from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps, ThunderstoreLinkProps {
+  queryParams?: string;
+  url: string;
+}
+
+const CyberstormLink = forwardRef<HTMLAnchorElement>(
```

Comment:
Given that this component does pretty much nothing, I see no reason to have it. The Link component below could as well include the `<a>` element directly.